### ISSUE TITLE
Normalizing Topic Frequency Based on the Number of Documents

### ISSUE THREAD

**Issue by nfsedaghat** [MaartenGr/BERTopic#1529]
Hi,

We have a similar number of daily documents in our application. We use BERTopic to create a 'topics over time' plot. When we create the model and plot in the first half of the month, topic frequency in the last month drops significantly compared to previous months. Waiting until month-end to create the model yields a more reasonable plot.

Do you recommend normalizing topic frequency by dividing it by the number of documents in the corresponding month? Your insights would be valuable.

**Comment by MaartenGr** [comment]
Yes, that would definitely help in interpreting the resulting frequencies. As it so happens, there is the `normalize_frequency` parameter in `.visualize_topics_over_time` that does just that.
